lib/appoptics/metrics/client.rb

```diff
@@ -46,6 +47,10 @@ def api_endpoint
       def api_endpoint=(endpoint)
         @api_endpoint = endpoint
       end
+      
+      def set_custom_headers(headers)
```

Comment:
The other set operators in this class appear to use the `foo=` method form (see `api_endpoint=` and `custom_user_agent=`). Any reason you decided to go with a `set_` method instead? I think consistency here would be nice.